website/docs/r/iothub_dps_certificate.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_iot_dps_certificate"
+sidebar_current: "docs-azurerm-resource-messaging-iot-dps_certificate"
+description: |-
+  Manages an IoT Device Provisioning Service Certificate.
+---
+
+# azurerm_iot_dps_certificate
```

Comment:
I think this file got mislabled? This looks like a copy of [iot_dps_certificate.html.markdown](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/r/iot_dps_certificate.html.markdown)

Reply:
whoops, i was wondering where these files ended up 🤣